lib/question.js

```diff
@@ -1072,6 +1072,12 @@ module.exports = {
                     locals.clientFilesCourseUrl = locals.urlPrefix + '/clientFilesCourse';
                     locals.clientFilesQuestionGeneratedFileUrl = questionUrl + 'generatedFilesQuestion/variant/' + locals.variant.id;
 
+                    locals.question.GHLink = null;
```

Comment:
I'm trying to keep variables like `locals.question` an exact copy of a row from the `questions` table, and not insert extra properties into them. In cases like this I'd prefer to put this into a new variable like `locals.questionGHLink`.

I'd also rather not have this in `lib/question.js` because it's only used in `instructorQuestion.ejs`. Instead, let's put this bit of code in `pages/instructorQuestion/instructorQuestion.js` (in the `.get()` handler).